### ISSUE TITLE
SSH Connection: Bad Local Forwarding Specification

### ISSUE THREAD

**Issue by rawkode** [sequelpro/sequelpro#2141]
I am getting this error when trying to connect to a MySQL server through a SSH tunnel. It works fine from the CLI when I replace the slashes with colons.

I am running OSX El Capitan, so I was wondering if that is perhaps using a different version of SSH?

```
 ~  /usr/bin/ssh -v -N -o ControlMaster=no -o ExitOnForwardFailure=yes -o ConnectTimeout=10 -o NumberOfPasswordPrompts=3 -i /Users/rawkode/.ssh/DevTech.pem -o TCPKeepAlive=no -o ServerAliveInterval=60 -o ServerAliveCountMax=1 ubuntu@some.ec2.box.com -L 61305/rds.box.amazonaws.com/3306

Bad local forwarding specification '61305/rds.box.amazonaws.com/3306'
```

**Comment by jmjones88** [comment]
I'm getting the error as well. The command works in the terminal if you change the '/' to : 

10.11
usage: ssh [-1246AaCfgKkMNnqsTtVvXxYy] [-b bind_address] [-c cipher_spec]
           [-D [bind_address:]port] [-E log_file] [-e escape_char]
           [-F configfile] [-I pkcs11] [-i identity_file]
           [-L [bind_address:]port:host:hostport] [-l login_name] [-m mac_spec]
           [-O ctl_cmd] [-o option] [-p port]
           [-Q cipher | cipher-auth | mac | kex | key]
           [-R [bind_address:]port:host:hostport] [-S ctl_path] [-W host:port]
           [-w local_tun[:remote_tun]] [user@]hostname [command]

Maybe there was some change in the syntax requirements of SSH? I was going to try to rebuild it from the Xcode project and find where that syntax is so I can get it working.

**Comment by willtcarey** [comment]
This is a duplicate of #2050.
I believe it's been fixed in the nightlies that you can get from http://nightly.sequelpro.com

**Comment by jmjones88** [comment]
Great, works for me. Thanks for the info!

**Comment by dmoagx** [comment]
Duplicate of #2050.
This is already fixed in 1.1rc1.

**Comment by rawkode** [comment]
Thanks - all sorted now :)
